app/models/contact_info.rb

```diff
@@ -11,18 +9,18 @@ class ContactInfo < ActiveRecord::Base
   before_validation :strip
 
   validates :user, presence: true
-  validates :type, presence: true, inclusion: { in: VALID_TYPES }
+  validates :type, presence: true
   validates :value,
             presence: true,
             uniqueness: {scope: [:user_id, :type]}
 
-  scope :email_addresses, where(type: 'EmailAddress')
+  scope :email_addresses, -> { where(type: 'EmailAddress') }
   sifter :email_addresses do type.eq 'EmailAddress' end
 
-  scope :verified, where(verified: true)
+  scope :verified, -> { where(verified: true) }
   sifter :verified do verified.eq true end
 
-  scope :with_users, lambda { includes(:user) }
+  scope :with_users, lambda { joins(:user) }
```

Comment:
Or actually I might just make it a `.joins(:user).eager_load(:user)` to force an inner join instead of an outer join...